const-oid/src/lib.rs

```diff
@@ -241,3 +241,9 @@ impl fmt::Display for ObjectIdentifier {
         Ok(())
     }
 }
+
+/// A trait expressing the association of a type with an OID.
+pub trait Typed {
```

Comment:
This could probably use a more descriptive name, but I'm having trouble thinking of a good one

Reply:
How about `OidValue`?